lldb/source/Plugins/SymbolFile/DWARF/SymbolFileDWARF.h

```diff
@@ -282,6 +282,11 @@ class SymbolFileDWARF : public SymbolFileCommon {
   bool GetSeparateDebugInfo(StructuredData::Dictionary &d,
                             bool errors_only) override;
 
+  // Gets a pair of loaded and total dwo file counts.
+  // For DWP files, this reports the counts for successfully loaded DWO CUs
```

Comment:
"For DWP files", do you mean "For split dwarf files"? Otherwise, you need to mention "dwo files" in addition to "dwp files".